src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$gap-size: 1px;
+$cells-in-row: 7;
+$height-cell: 100px;
+$width-cell: 100px;
+$week-days:
+  'mon' 1,
+  'tue' 2,
+  'wed' 3,
+  'thu' 4,
+  'fri' 5,
+  'sat' 6,
+  'sun' 7;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), variables that are used in multiple places should be placed in a separate file `_variables.scss` and then imported where needed.